twined/schema/children_schema.json

```diff
@@ -8,15 +8,34 @@
                 "type": "string"
             },
             "id": {
-                "description": "The universally unique ID of the running child twin",
+                "description": "The universally unique ID (UUID) of the running child twin",
                 "type": "string"
             },
-            "uri_env_name": {
-                "description": "Name of the environment variable containing the URI of the twin (which may contain credentials)",
-                "type": "string",
-                "pattern": "^[A-Z]+(?:_[A-Z]+)*$"
+            "backend": {
+                "description": "The backend running the child.",
+                "type": "object",
+                "oneOf": [
+                    {
+                        "type": "object",
+                        "title": "GCP Pub/Sub",
+                        "properties": {
+                            "name": {
+                                "description": "Type of backend (in this case, it should be GCPPubSub",
+                                "type": "string"
+                            },
```

Comment:
You could set the pattern of this to be a regex to enforce "GCPPubSub"
https://json-schema.org/draft-07/json-schema-validation.html#rfc.section.6.3.3